src/shapes/textbox.class.ts

```diff
@@ -457,11 +457,16 @@ export class Textbox extends IText {
   }
 }
 
+// @TODO: Many things here are configuration related and shouldn't be on the class nor prototype
+// regexes, list of properties that are not suppose to change by instances, magic consts.
+// this will be a separated effort
 export const textboxDefaultValues: Partial<TClassProperties<Textbox>> = {
   type: 'textbox',
   minWidth: 20,
   dynamicMinWidth: 2,
+  // with the new controls this noScaleCache here shouldn't be needed anymore
   lockScalingFlip: true,
+  // with the new controls this noScaleCache here shouldn't be needed anymore
```

Comment:
care to elaborate?

Reply:
I think i was thinking at the width behaviour and that scaling does not affect textbox because we usually change the width. but i think i forgot of both axis scaling